src/test/java/org/reaktivity/specification/nukleus/http/streams/server/rfc7230/newsystem/ArchitectureIT.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.specification.nukleus.http.streams.server.rfc7230.newsystem;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.rules.RuleChain.outerRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.k3po.junit.annotation.ScriptProperty;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+import org.reaktivity.specification.nukleus.NukleusRule;
+
+public class ArchitectureIT
+{
+    private final K3poRule k3po = new K3poRule()
+        .addScriptRoot("streams", "org/reaktivity/specification/nukleus/http/streams/rfc7230/architecture");
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(5, SECONDS));
+
+    private final NukleusRule nukleus = new NukleusRule()
```

Comment:
This will will be changing, probably to be different for the raw and cooked cases.